src/java/org/apache/cassandra/tcm/PaxosBackedProcessor.java

```diff
@@ -136,7 +138,17 @@ public ClusterMetadata fetchLogAndWait(Epoch waitFor, Retry.Deadline retryPolicy
             while (iter.hasNext())
             {
                 FetchLogRequest request = iter.next();
-                if (request.condition.awaitUninterruptibly(Math.max(0, nextTimeout - Clock.Global.nanoTime()), TimeUnit.NANOSECONDS) &&
+                boolean awaitResult;
```

Comment:
filed CASSANDRA-19761, this is a TCM issue and Sam said he will take a look... my work is kinda a hack more than anything, I just wanted to unblock my tests!